src/components/ProviderInitializer.tsx

```diff
@@ -67,6 +68,8 @@ export function ProviderInitializer() {
   const initializedAccountRef = useRef(false);
   const dispatch = useDispatch();
 
+  useWebViewLogin();
```

Comment:
I'd use a condition here to not use this hook for all dApp, just if you need this. The condition cand be added on DappProvider